lisa/sut_orchestrator/libvirt/context.py

```diff
@@ -48,6 +48,7 @@ class NodeContext:
     vm_name: str = ""
     firmware_source_path: str = ""
     firmware_path: str = ""
+    guest_vm_type: str = ""
```

Comment:
Please make this an enum type instead of string. That way, it is well defined what the possible values for `guest_vm_type` are.

Reply:
The ```guest_vm_type``` will take value from security profile mapping which has a list of possible types of guest vm and returns a string literal of guest vm type.

Reply:
The security profile mapping could use the enum as well instead of string literals.

Let's make it a direct enforcement of possible values by defining an enum. That way someone reading the code can easily figure out the possible values by just looking at the type of the `guest_vm_type` field. Otherwise, they would need to do some digging around to find the relevant code in features.py...

Reply:
Sure, will update the security profile mapping accordingly.